topics/single-cell/tutorials/alevin-commandline/tutorial.md

```diff
@@ -237,6 +256,18 @@ Now load the library that we have previously installed in terminal:
 library(tximeta)
 ```
 
+> <comment-title>"There is no package called ‘DropletUtils’"?</comment-title>
+> If you are using JupyterLab Notebook {% icon tool-versions %} **version 1.0.1** and you encounter this error message while loading the library, you can run the code below. Beware, it can take up to 25 minutes to run!
+> ```bash
```

Comment:
Can you install your packages inside the Jupyter kernel with `!conda install ...` that way you do not need to change the kernels so often.

Reply:
Or install in the bash kernel all packages into the environment of the python kernel that should also work.

Happy to discuss and help that in chat, please ping me.

Reply:
Can install in the bash kernel packages into the environment of R kernel? 

The fix for now is using installation by BiocManager when I switch from bash to R. But it's quite long for those two packages

Reply:
I would need to understand what you want to do ... but you can install packages into any conda environment from any kernel, you just need to use adopted `conda ` commands.

For example:

## version 1

- change to bash
- conda activate the-python-env
- conda install whatever you want
- going back to jupyter kernel
- done

## version 2

- stay in the jupyter kernel
- install via jupyter magic `!conda install ...`
- done

Reply:
If I understood correctly:
Version 1: installing the packages into python environment while being in bash - doesn't help here because we are switching from bash to R and not python
Version 2: magic command `!conda install ` works in python environment, not in R. This is actually the trick I used in Google Colab notebook since there is no bash kernel, only python or R.
So based on what I tested, I just went for installing the packages that are required in the "R part" of the tutorial by using BiocManager::install() while already being in R kernel - works but takes a while. If those packages were pre-installed in the notebook, that would probably solve the problem of long installation by BiocManager::install()

Reply:
Ok, if you are already using the R kernel, you can also change to the r virtual env.

Reply:
Is this section now still needed?

Reply:
I changed this comment to make it more general so that it doesn't mention the version of the notebook. I would keep it as advice for anyone who might be using this tutorial to run the code locally and would encounter such a problem.